express/blocks/image-list/image-list.css

```diff
@@ -0,0 +1,96 @@
+main .section:has(.image-list) > div {
+  margin: auto;
```

Comment:
darn it , I've been missing these things. Don't do this :) 
no changes to the section styles from within the block .